tests/main/selinux-lxd/task.yaml

```diff
@@ -0,0 +1,59 @@
+summary: Check that LXD snap works on a system using SELinux in enforcing mode
+
+description: |
+    Make sure that LXD snap can be installed and used on a SELinux enable
+    system, with enforcing mode on.
+
+# Systems with SELinux enabled out of the box.
+systems: [fedora-*, centos-*]
+
+# lxd downloads can be quite slow
+kill-timeout: 25m
+
+# Start before anything else as it can take a really long time.
+priority: 1000
```

Comment:
Why do we want to start it with higher prio?

Reply:
Sam motivation as here https://github.com/snapcore/snapd/pull/6711#discussion_r274840547